### PR TITLE
Olubimi Michael added a new card

### PR DESCRIPTION
**Pull request by olumide12-cell**
New card added by me (olubimi Michael)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3604,6 +3604,37 @@ <h2>A project for first-time contributions</h2>
...
+        <div class="card">
+          <p class="name">Olubimi michael olumide</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/olumide12-cell/" target="_blank">GitHub</a>
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/olubimi-michael/" target="_blank">LinkedIn</a>
+          </p>
+          <p class="about">Hi, I'm Olumimi Michael Olumide, a full stack software engineer enthusiast, 
+            currently a software engineer student at ALX, a MERN stack web developer. I'm currently seeking a remote position; message me on LinkedIn and check out my repos to see what I've got to offer!</p>
```
Comment (Michaelin007) [inline index.html:3618]: Can you review the about. I'm not sure if it's permitted to advertise yourself for a remote job. or is it permitted @Syknapse

**Comment by olumide12-cell** [comment]
Thank you for thee feedback, I will do the correction instead and push
again..

Thanks

File: index.html [hunk 2]
```diff
@@ -3604,6 +3604,35 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Mariana Martins' card END ________  -->
 
+        
+        <!-- ________ Michael card START ________  -->
+        <div class="card">
+          <p class="name">Olubimi Michael Olumide</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/olumide12-cell/" target="_blank">GitHub</a>
```
Comment (TenTraicion) [inline index.html:3613]:
```suggestion
            <a href="https://github.com/olumide12-cell" target="_blank">@olumide12-cell</a>
```

File: index.html [hunk 3]
```diff
@@ -3604,6 +3604,35 @@ <h2>A project for first-time contributions</h2>
...
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/olubimi-michael/" target="_blank">LinkedIn</a>
```
Comment (TenTraicion) [inline index.html:3615]:
```suggestion
            <a href="https://www.linkedin.com/in/olubimi-michael" target="_blank">@olubimi-michael</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@olumide12-cell, Thanks for the contribution. but you need to change your social handle as mentioned below and move your card to the top of card section (after line 117). 
Kindly make the changes and commit. Thanks.
for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF).

**Comment by CandidDeer** [comment]
Thank you for your contribution and making the changes. I have merged your PR now and your card will be live in a moment.

**Pull request merged**